nemo/collections/asr/modules/conv_asr.py

```diff
@@ -62,8 +63,14 @@ def _prepare_for_export(self, **kwargs):
         m_count = 0
         for m in self.modules():
             if isinstance(m, MaskedConv1d):
-                m.use_mask = False
-                m_count += 1
+                if hasattr(self, '_rnnt_export') and self._rnnt_export:
```

Comment:
Why not overriding this function in _rnnt_export ?

Reply:
That's an option, but that would make framework dependent on what type of encoder was used. This way the Conformer and Citrinet can chose to supply seperate input-output ages for CTC and RNNT exports while CTC and RNNT export framework remains unaware of what encoder type it is

Reply:
Also, I do not see any changes in this PR (nor in main) that would set _rnnt_export attribute - are those part of another PR ?

Reply:
Part of this PR Inside the asr_model.py file (needs to be expanded)